.github/workflows/release.yml

```diff
@@ -0,0 +1,130 @@
+# MIT License
+#
+# Copyright (c) 2020 Yaroslav Bolyukin
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
```

Comment:
I think this is not the expected workflow behavior, it'll be triggered by pushs on `master` ___OR___ any tags with `v` as they prefix!
I recommend the use of `release` instead:
```yml
on:
  release:
    types: released
```

Reply:
This [line](https://github.com/bruno-delfino1995/kct/blob/release/.github/workflows/release.yml#L50) prevents the workflow to run when something is pushed to the `master` branch!
Tks @bruno-delfino1995

Reply:
That's exactly what I want. The release should only run for tags on master. However, I don't know if that is feasible because `github.ref` is equal to what triggered the run. I don't think that something like [this](https://stackoverflow.com/questions/59874409/trigger-github-workflow-only-if-pushed-to-specific-branch-and-tag-exists) would work, but since I want to create a release only for tags, the master branch as a trigger for this is useless. However, I can't use the release event because that's what this workflow will be doing; it'd conflict with my manual release.